### PR TITLE
[Task] - Create register UI

### PR DESCRIPTION
**Pull request by cmvn-linhnguyen**
### References
- [Sprint backlog URL](https://docs.google.com/spreadsheets/d/15HwUJ5TIas9ywzYUvKoW8NMbrCrGb9qP/edit#gid=1890329971)
- Task: Create Register Page UI

### Description
- Create Register Page UI

### Screenshot
image

### Checklist
- [x] App run correctly
- [x] The code has been tested in the local environment.
- [x] No convention errors

### CODE REVIEW

File: src/app/core/auth/containers/Register.tsx [hunk 1]
```diff
@@ -1,8 +1,90 @@
...
+                <InputField label="Password" type="password" id="password" />
+              </div>
+              <div className="col col-6">
+                <InputField
+                  label="Confirm Password"
+                  type="password"
+                  id="confirm-password"
+                />
+              </div>
+              <div className="col col-12 d-flex register-action">
```
Comment (cmvn-thaile) [inline src/app/core/auth/containers/Register.tsx:67]: không để chung

File: src/app/core/auth/containers/Register.tsx [hunk 2]
```diff
@@ -1,8 +1,90 @@
...
+          <div className="register-redirect">
+            <span className="register-redirect-message">
+              Already have an account?{' '}
+            </span>
+            <a className="register-redirect-link" href="#">
+              Login
+            </a>
+          </div>
+        </div>
+        <div className="col col-5 register-cover">
```
Comment (cmvn-thaile) [inline src/app/core/auth/containers/Register.tsx:83]: không để chung

File: src/app/shared/components/InputField.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import React, { InputHTMLAttributes, useState } from 'react';
+
+interface InputFieldProps extends InputHTMLAttributes<HTMLInputElement> {
+  label: string;
+  error?: string;
+}
+
+export const InputField: React.FC<InputFieldProps> = ({
```
Comment (st-tiendinh) [inline src/app/shared/components/InputField.tsx:8]: Nên đặt tên component là InputGroup theo Bootstrap

File: src/app/shared/components/InputField.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  error,
+  className,
+  type,
+  ...rest
+}) => {
+  const hasError = !!error;
+  const [isShowPassword, setIsShowPassword] = useState(false);
+
+  return (
+    <div className={`input-field ${hasError ? 'error' : null}`}>
```
Comment (st-tiendinh) [inline src/app/shared/components/InputField.tsx:21]: Đổi tên className

File: src/stylesheet/modules/_input-field.scss [hunk 5]
```diff
@@ -0,0 +1,94 @@
+.input-field {
```
Comment (st-tiendinh) [inline src/stylesheet/modules/_input-field.scss:1]: Đổi tên className

**Pull request merged**